jekyll/_cci2/gitlab-trigger-options.adoc

```diff
@@ -10,8 +10,57 @@ contentTags:
 :icons: font
 :experimental:
 
-GitLab **Trigger filters** allow you to determine when a trigger should initiate a build based on the parameters provided by Gitlab’s webhook. CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
+GitLab **Trigger filters** allow you to determine when a trigger should start a build based on the parameters provided by GitLab's webhook.
+
+CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
+
+== Introduction
+
+CircleCI projects have pipelines and triggers.
+
+* Pipelines define the various CI/CD 'units' of work that make up your project.
+* Triggers define when and how to start a pipeline.
+
+== Trigger options
+
+Configure triggers under menu:Project settings[Triggers]. You can select one of the following filters:
+
+.Trigger options under Project settings
+image::triggers/gitlab-cloud-trigger-filter-only-mr.png[Trigger details]
+
+* **Build Push and Merge Requests**: pipelines start when you push commits to the repository, regardless of whether the branch you push commits to has an open merge request.
+* **Only Merge Requests**: pipelines start **only** if you push commits to a branch that has an open merge request.
+* **Customize Trigger Filters**: pipelines start when the event matches the custom filters.
+
+== Custom trigger filters
+
+.Custom Trigger Filters
+image::triggers/gitlab-trigger-customize-edit.png[Custom trigger filters]
+
+Each filter follows a specific format:
+
+1. Parameter name.
+2. Logical operator.
+3. Parameter value.
+
+[TIP]
+====
+Use the btn:[Add Filter] button to configure multiple filters. +
+Then specify the `and` or `or` relationship between all configured filters.
+====
+
+=== Available parameters
+
+* **EventType**: the type of event that occurred in GitLab. It can either be `Push`, `Merge Request`, or `Tag Push`.
+* **Branch**: the name of the branch or a regular expression to match against the branch name.
+* **Tag**: the name of the tag or a regular expression to match against the tag name.
+* **Is Fork Merge Request**: whether the merge request is a fork merge request. The value can either be `true` or `false`.
+
+=== Available logical operators
+
+* `= (equals)`
+* `!= (not equals)`
+* `matches (regex)`
+* `not matches (regex)`
```

Comment:
I understand why you have done this, to match the UI, but it seems odd to me to have the explainers inside the backticks. Could you use the same format as above and use a colon to show "thing: definition"? Or if that also looks off just keep the explainers in the parentheses but not in the backticks?